src/coinbase/staking_balance.ts

```diff
@@ -22,15 +22,15 @@ export class StakingBalance {
    *
    * @param networkId - The network ID.
    * @param assetId - The asset ID.
-   * @param address - The address ID.
+   * @param addressId - The address ID.
    * @param startTime - The start time.
    * @param endTime - The end time.
    * @returns The staking balances.
    */
   public static async list(
     networkId: string,
     assetId: string,
-    address: string,
+    addressId: string,
     startTime: string,
     endTime: string,
```

Comment:
Can we make this take an object v.s. parameters?

6 parameters is quite a big to have contextual ordering for